pkg/integration/system_intakes_test.go

```diff
@@ -91,6 +91,58 @@ func (s IntegrationTestSuite) TestSystemIntakeEndpoints() {
 		s.Equal(http.StatusOK, resp.StatusCode)
 	})
 
+	// TODO uncomment when we have CircleCI configured to hit CEDAR
```

Comment:
You can also flag this on local environment so it still has to run in our dev environments.